scipy/linalg/flapack.pyf.src

```diff
@@ -708,6 +708,34 @@ interface
 
    end subroutine <prefix2>gelss
 
+    subroutine <prefix2>gelss_lwork(m,n,minmn,maxmn,nrhs,a,b,s,cond,r,work,lwork,info)
+
+   ! work,info = gelss_lwork(m,n,nrhs,cond=-1.0)
+   ! Query for optimal lwork size
+
+     fortranname <prefix2>gelss
+     callstatement (*f2py_func)(&m,&n,&nrhs,&a,&m,&b,&maxmn,&s,&cond,&r,&work,&lwork,&info)
+     callprotoargument int*,int*,int*,<ctype2>*,int*,<ctype2>*,int*,<ctype2>*,<ctype2>*,int*,<ctype2>*,int*,int*
+
+     integer intent(in):: m 
+     integer intent(in):: n
+     integer intent(hide),depend(m,n):: minmn = MIN(m,n)
```

Comment:
Yes, only in ?gelss.  In ?gelss_lwork, ?gelsy_lwork and ?gelsy it is unused. If its used fine, but at present it isn't used at all in several places.